sunpy/net/base_client.py

```diff
@@ -206,6 +206,43 @@ def _print_client(client, html=False):
     return '\n'.join(lines)
 
 
+def _repr_client(client, html=False):
```

Comment:
I am unsure why we need a new function, can you explain?

Reply:
I just thought that if in case the user wanted to display the entire table they could use _print_client and also I was not sure if _print_client was called someplace else. Should I add the visible_entries parameter in _print_client itself?

Reply:
Why not just add a new arg to the print client function to limit the line length?

Reply:
Yup, that sounds better. I'll do that.